plugins/extractors/postgres/postgres.go

```diff
@@ -230,10 +232,22 @@ func isNullable(value string) bool {
 
 // connection generates a connection string
 func connection(cfg Config, database string) (db *sql.DB, err error) {
-	connStr := fmt.Sprintf("postgres://%s:%s@%s/%s?sslmode=disable", cfg.UserID, cfg.Password, cfg.Host, database)
+	connStr := fmt.Sprintf("%s/%s?sslmode=disable", cfg.ConnectionURL, database)
 	return sql.Open("postgres", connStr)
 }
 
+// splitURL splits the connection string into the host
+func splitURL(cfg Config) (host string, err error) {
```

Comment:
The function does not really `split` the URL. It is actually just `fetching` host from Connection URL. I think the proper name would be something align with `GetHostFromConnectionURL`.